### PR TITLE
Added all the content of the middle section following the Figma design

### PR DESCRIPTION
**Pull request by jaimevillegas**
I have made modifications to the files, sticking with the Figma design. Now, the web page has a showcase of works and an About Me section.

Changes Made:
- Modified index.html
- Modified style.css
- Added icons

Thank you a lot for taking the time to review my code and suggest any changes you consider.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,6 +60,166 @@ <h2 class='highlited-title'>I'm a software developer</h2>
...
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <button>See Project</button>
+      </div>
+    </section>
+  </section>
```
Comment (Ibizugbe) [inline index.html:169]: I think everything here should be wrapped in one `<section>` tag. this is meant to be the work section as described in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). kindly wrap everything relating to your "works" in one `<section>` tag🤗.
You can put  `<section class='recent-works'>` inside `<section class="projects">`😉

File: index.html [hunk 2]
```diff
@@ -60,6 +60,166 @@ <h2 class='highlited-title'>I'm a software developer</h2>
...
+      <h2>Skills</h2>
+      <ul>
+        <li>Codekit</li>
+        <li>GitHub</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
   </section>
```
Comment (Ibizugbe) [inline index.html:223]: I think everything here should be wrapped in one `<section>` tag. this is meant to be the about section as described in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). kindly wrap everything about you in one `<section>` tag🤗.
You can put  `  <section class="habilities">` inside `<section class="about-me-container">`😉 don't forget, you're meant to have just two major sections in this milestone. `work` and `about` sections😉

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @jaimevillegas 👋, 

Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅
- [ ] Linters are passing👍.
- [ ] Pull request has a descriptive header and summary👍.
- [ ] Nice work sticking to the design👍.

## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jaimevillegas** [comment]
Hello @Ibizugbe! Thank you for taking the time to kindly review my code.

I have made the following changes according to your suggestions:

- I have now 2 main sections: `Projects` and `about-me-containter` making it easier to read and follow.

Thank you again for your feedback. I'm gonna make a new Review request in order to be able to merge this branch.

Regards

**Review by bienvenuushindi — APPROVED** [review]
Hi @jaimevillegas    , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**